wp-content/themes/csisnuclearnetwork/assets/_scss/pages/home.scss

```diff
@@ -494,6 +495,164 @@
+      padding-top: rem(72);
+    }
+
+    @include breakpoint(medium) {
+      grid-template-areas: 
+        'event twitter'
+        'news twitter';
+      grid-template-columns: 1fr 1fr;
+      grid-template-rows: max-content;
+    }
+
+    @include breakpoint(xlarge) {
+      grid-template-areas: 
+        'event'
+        'news'
+        'twitter';
+      grid-template-columns: 1fr;
+    }
+
+    .post-meta__event-meta {
+      clear: both;
+      margin: 0 -6vw;
+      margin-top: rem(40);
+      padding: 0;
+      border-top: 3px $color-border-light-115 solid;
+
+      @include breakpoint(medium) {
+        margin-right: 0;
+        margin-left: -6vw;
+      }
+
+      @include breakpoint(xlarge) {
+        display: block;
+        margin: 0 -6%;
+        margin-top: rem(40);
+      }
+    }
+
+    &::after,
+    &::before {
+      z-index: -1;
+    }
+
+    .twitter-section {
+      grid-area: twitter;
+
+      &-title {
+        margin-top: rem(32);
+        margin-bottom: rem(16);
+        @extend %text-style-heading-large-strong;
+
+        @include breakpoint(medium) {
+          margin-top: 0;
+        }
+
+        @include breakpoint(xlarge) {
+          margin-top: rem(32);
+        }
+      }
+    }
+
+    .upcoming-event {
+      grid-area: event;
+
+      &-title {
+        @extend %text-style-heading-large-strong;
+        margin-bottom: rem(16);
+      }
+    }
+
+    .post-block {
+      --post-meta: #{$color-white-100};
+      padding: 0;
+      padding-top: rem(16);
+      // background-color: unset;
+
+      .post-meta {
+        --post-meta: #{$color-accent-secondary-yellow-100};
+      }
+
+      &__title {
+        color: var(--post-meta);
+        @extend %text-style-heading-medium;
+      }
+    }
+
+    .news-posts {
+      grid-area: news;
+      margin-top: rem(32);
+
+      &-title {
+        margin-bottom: rem(16);
+        @extend %text-style-heading-large-strong;
+      }
+
+      .post-meta__event-meta {
+
+        @include breakpoint(medium) {
+          display: none;
+        }
+
+        @include breakpoint(xlarge) {
+          display: block;
+        }
+      }
+
+      /* stylelint-disable */
+      .post-block-related {
+        text-align: left;
+
+        &__title {
+          @extend %text-style-heading-medium;
+          color: $color-white-100;
+
+            &:hover {
+              color: $color-accent-secondary-yellow-100;
+            }
+
+            &:focus {
+              background-color: $color-accent-primary-blue-100;
+            }
+        }
+
+        .post-meta__date {
+          .post-meta__label {
+            display: none;
+          }
+        }
+
+        .post-meta__terms {
+          --post-meta: #{$color-accent-secondary-yellow-100};
+        }
+
+        .post-meta__date {
+          --post-meta: #{$color-white-180};
+          @extend %text-style-label-medium;
+        }
+        /* stylelint-enable */
+      }
+
+      .post-block-related:nth-of-type(2) {
```

Comment:
Nice! In case they change the number of posts shown, let's change this to `.post-block-related:not(:first-child):not(:last-child)`.